.github/workflows/cpppackage.yml

```diff
@@ -0,0 +1,51 @@
+name: C++ package
+
+on: [push, pull_request]
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Debug
```

Comment:
Shouldn't this be Release by default?

Reply:
I've set tests to build when build type is `Debug`, so for the workflow this is correct.